sdk/purview/azure-purview-account/tests/test_smoke.py

```diff
@@ -0,0 +1,26 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+from testcase import PurviewAccountTest, PurviewAccountPowerShellPreparer
+from _util import PurviewAccountRecordingProcessor, PurviewAccountCollectionsRecordingProcessor
+
+
+class PurviewAccountSmokeTest(PurviewAccountTest):
+
+    @PurviewAccountPowerShellPreparer()
+    def test_basic_smoke_test(self, purviewaccount_endpoint):
+        self.recording_processors.append(PurviewAccountRecordingProcessor())
+        client = self.create_client(endpoint=purviewaccount_endpoint)
+        response = client.accounts.get_access_keys()
+        assert set(response.keys()) == set(['atlasKafkaPrimaryEndpoint', 'atlasKafkaSecondaryEndpoint'])
+
+    @PurviewAccountPowerShellPreparer()
+    def test_collections_list(self, purviewaccount_endpoint):
+        self.recording_processors.append(PurviewAccountCollectionsRecordingProcessor())
+        client = self.create_client(endpoint=purviewaccount_endpoint)
+        response = client.collections.list_collections()
+        result = [item for item in response]
+        assert len(result) > 0
```

Comment:
I'd also verify some of the attributes in the key response if possible, for example verify that their names are what you expect etc

Reply:
side note: i refer to "key names", not 100% sure that's actually whats in the response body, but what I mean is just identifying information about each key

Reply:
I think detailed check is not necessary since the resource is created manually in advance. And the resource may be changed in the future. But we could check whether keys exist.